### PR TITLE
EPMDJ-8194 Add metada to TestInfo

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: admin-part/src/main/kotlin/BuildTests.kt [hunk 1]
```diff
@@ -36,37 +37,40 @@ internal fun BundleCounters.testsWith(
...
     TypedTest(name, type) in bundleCounters.byTest
 }
 
 internal fun CoverContext.testsToRunDto(
-    bundleCounters: BundleCounters = build.bundleCounters
+    bundleCounters: BundleCounters = build.bundleCounters,
 ): List<TestCoverageDto> = testsToRun.flatMap { (type, tests) ->
-    tests.map { name ->
-        val typedTest = TypedTest(type = type, name = name)
+    tests.map { test ->
```
Comment (Mazel-Tovr) [inline admin-part/src/main/kotlin/BuildTests.kt:67]:
```suggestion
    testData.map { test ->
```

File: api/src/commonMain/kotlin/Model.kt [hunk 2]
```diff
@@ -71,6 +71,13 @@ data class TestInfo(
     val result: TestResult,
     val startedAt: Long,
     val finishedAt: Long,
+    val metadata: String = "",
+)
+
+@Serializable
+class TestData(
+    val name: String,
+    val metadata: String,
```
Comment (Mazel-Tovr) [inline api/src/commonMain/kotlin/Model.kt:80]: map<String,String>

File: api/src/commonMain/kotlin/Model.kt [hunk 3]
```diff
@@ -297,15 +298,32 @@ data class TestCoverageDto(
...
 
+@Serializable
+data class TestMetadata(
+    val hash: String = "",
+    val data: Map<String, String> = emptyMap(),
+){
+    companion object {
+        val emptyMetadata = TestMetadata()
+    }
+}
```
Comment (LikhanovMaxim) [inline api/src/commonMain/kotlin/Model.kt:319]: is a need to add JvmSerializable?

File: admin-part/src/main/kotlin/Session.kt [hunk 4]
```diff
@@ -65,7 +61,7 @@ class ActiveSession(
         probe.id?.let { probe } ?: probe.copy(id = probe.id())
     }.forEach { probe ->
         if (true in probe.probes) {
-            val typedTest = probe.testName.weakIntern().typedTest(testType.weakIntern())
+            val typedTest = probe.testName.typedTest(testType)
```
Comment (LikhanovMaxim) [inline admin-part/src/main/kotlin/Session.kt:64]: type will be repeated, why did you remove testType.weakIntern()?

File: gradle.properties [hunk 5]
```diff
@@ -20,5 +20,5 @@ licenseVersion = 0.15.0
 benchmarkVersion=0.3.0
 kotlinNoarg=1.4.21
 
-drillApiVersion=0.8.0-0
+drillApiVersion=0.8.0-1
```
Comment (LikhanovMaxim) [inline gradle.properties:23]: why did you increase?

File: api/src/commonMain/kotlin/Model.kt (hunk 3 above)
Comment (Mazel-Tovr) [reply api/src/commonMain/kotlin/Model.kt:319]: we should remove JvmSerializable from every classes, because we are no longer use fst serialization

File: admin-part/src/main/kotlin/Session.kt (hunk 4 above)
Comment (Mazel-Tovr) [reply admin-part/src/main/kotlin/Session.kt:64]: we use .weakIntern in ``String.typedTest(String)```  method

File: gradle.properties (hunk 5 above)
Comment (Mazel-Tovr) [reply gradle.properties:23]: so the tests don't fail

**Review by LikhanovMaxim — APPROVED** [review]
